Taskfile.yml

```
@@ -28,12 +28,31 @@ tasks:
       - task: docs:gen:commands
       - task: docs:gen:protobuf
 
+  docs:patch-theme:
+    desc: Downloads documentation theme and patch it with custom header and footer
+    cmds:
+      - |
+        [[ -d mkdocs-material ]] && git clone https://github.com/squidfunk/mkdocs-material.git -b 5.5.14 --depth 1
+        # Ugly hack so that this doesn't fail if patch is already applied
+        git apply --check mkdocs-material-patches/*.patch && git apply mkdocs-material-patches/*.patch; echo
```

```suggestion
        git apply --check mkdocs-material-patches/*.patch && git apply mkdocs-material-patches/*.patch; true
```
Maybe this way is a bit more self-explanatory?

---

Oh, yeah! Didn't think about this, thanks Per!